sanssouci/lambda_calibration.py

```diff
@@ -88,6 +88,53 @@ def get_perm_p(X, categ, B=100, row_test_fun=stats.ttest_ind):
     return pval0
 
 
+def get_permuted_p_values_one_sample(X, B=100):
+    """
+    Get permutation p-values: Get a matrix of p-values under the null
+    hypothesis obtained by sign-flipping (one-sample test).
+
+    Parameters
+    ----------
+
+    X : array-like of shape (n,p)
+        numpy array of size [n,p], containing n observations of p variables
+        (hypotheses)
+    B : int
+        number of sign-flippings to be performed (default=100)
+
+    Returns
+    -------
+
+    pva0 : array-like of shape (B, p)
+        A numpy array of size [B,p], whose entry i,j corresponds to
+        p_{(j)}(g_i.X) with notation of the AoS 2020 paper cited below
+        (section 4.5) [1]_
```

Comment:
The array is sorted, right ? Please make it clear in the docstring.

Reply:
Thx, added in commit e4b7824538533c2400415a7844cfb5306f5c8149 (+ 86e6c9f885cf7410f6ccdc0741c48031b44c54c4 which corrects a wrong addition)